rpcs3/Emu/Cell/lv2/sys_prx.cpp

```diff
@@ -786,7 +786,7 @@ error_code _sys_prx_get_module_id_by_name(ppu_thread& ppu, vm::cptr<char> name,
 
 	//if (realName == "?") ...
 
-	return CELL_PRX_ERROR_UNKNOWN_MODULE;
+	return not_an_error(CELL_PRX_ERROR_UNKNOWN_MODULE);
```

Comment:
Why change this?

Reply:
We have this function logged already and it's not an error.

Reply:
where

Reply:
line 785

Reply:
the error is still a valid error though. so why silence it.

Reply:
It's not error in this syscall and it's complex to explain it.